ooniprobe/View/RunTestsViewController.mm

```diff
@@ -75,6 +76,8 @@ - (IBAction)runTests:(id)sender event:(id)event {
     CGPoint currentTouchPosition = [[[event allTouches] anyObject] locationInView:self.tableView];
     NSIndexPath *indexPath = [self.tableView indexPathForRowAtPoint: currentTouchPosition];
     NetworkMeasurement *current = [currentTests.availableNetworkMeasurements objectAtIndex:indexPath.row];
+    //TODO set test list
+    current.inputs = nil;
```

Comment:
Does this mean that we still need to do work to finish this release, or is this in preparation for future changes?

Reply:
The second thing you said

Reply:
Ok